docs/guides/migrating-from-camunda-7/data-migrator/cockpit-plugin.md

```diff
@@ -0,0 +1,103 @@
+---
+id: cockpit-plugin
+title: Cockpit plugin
+sidebar_label: Cockpit plugin
+description: "Web-based interface for viewing information about skipped entities during the migration process."
+---
+
+:::warning Experimental feature
+The Cockpit plugin is an **experimental** feature and should be used with caution in production environments.
+:::
+
+The Cockpit plugin provides a web-based interface for viewing information about skipped entities during the migration process. It integrates with Camunda 7 Cockpit to give you visibility into which process instances, variables, or other entities were skipped during migration and the reasons why.
+
+For more information on Camunda 7 plugins, see the [Camunda 7 documentation](https://docs.camunda.org/manual/latest/webapps/cockpit/extend/plugins/).
+
+## Prerequisites
+
+To use the Cockpit plugin, run the migrator with the following setting:
+
+```yaml
+camunda.migrator:
+  save-skip-reason: true
```

Comment:
/change This flag should be documented in our properties reference as well.

Reply:
Added to [config-properties.md](https://github.com/camunda/camunda-docs/pull/6717/commits/a86a5850c9848e9b02e353b553b65c1e1b446937#diff-f86c10e7e7e7a07d2b63fd1a511cacbc145c70443edc445d008d91f4b1fb0775)